packages/reactivue/src/useSetup.ts

```diff
@@ -53,12 +91,18 @@ export function useSetup<State, Props = {}>(
       watch(
         data,
         () => {
-          useInstanceScope(id, () => {
-            invokeLifeCycle(LifecycleHooks.BEFORE_UPDATE)
-            // trigger React update
-            setTick(+new Date())
-            invokeLifeCycle(LifecycleHooks.UPDATED)
-          })
+          /**
+           * Prevent triggering rerender when component
+           * is about to unmount or really unmounted
+           */
+          if (!instance.willUnmount) {
```

Comment:
```suggestion
          if (instance.isUnmounting || instance.isUnmounted)
            return
```

Early return feels more intuitive here

Reply:
`instance.isUnmounted` unnecessary in here. All effects will be cleared before this value become true. 

Probably you added this because of _"or really unmounted"_ comment. I mean here is the components removed by a value. {{show && <Comp />}}. That will be really unmounting component if show variable is falsy. But its reactivity effects will be alive, because we postponed removing effects in development.